spec/Section 6 -- Execution.md

```diff
@@ -506,25 +530,39 @@ CollectFields(objectType, selectionSet, variableValues, visitedFragments):
       * If {DoesFragmentTypeApply(objectType, fragmentType)} is false, continue
         with the next {selection} in {selectionSet}.
       * Let {fragmentSelectionSet} be the top-level selection set of {fragment}.
+      * If {fragmentSpreadName} provides the directive `@defer`, let {deferDirective} be that directive.
+        * If {deferDirective}'s {if} argument is {true} or is a variable in {variableValues} with the value {true}:
+          * Let {deferredFragment} be the result of calling {DeferFragment(objectType, objectValue, fragmentSelectionSet, parentPath)}.
+          * Append {deferredFragment} to {deferredFragments}.
+          * Continue with the next {selection} in {selectionSet}.
       * Let {fragmentGroupedFieldSet} be the result of calling
         {CollectFields(objectType, fragmentSelectionSet, variableValues, visitedFragments)}.
       * For each {fragmentGroup} in {fragmentGroupedFieldSet}:
         * Let {responseKey} be the response key shared by all fields in {fragmentGroup}.
         * Let {groupForResponseKey} be the list in {groupedFields} for
           {responseKey}; if no such list exists, create it as an empty list.
         * Append all items in {fragmentGroup} to {groupForResponseKey}.
+      * Add {fragmentSpreadName} to {visitedFragments}.
     * If {selection} is an {InlineFragment}:
       * Let {fragmentType} be the type condition on {selection}.
       * If {fragmentType} is not {null} and {DoesFragmentTypeApply(objectType, fragmentType)} is false, continue
         with the next {selection} in {selectionSet}.
       * Let {fragmentSelectionSet} be the top-level selection set of {selection}.
-      * Let {fragmentGroupedFieldSet} be the result of calling {CollectFields(objectType, fragmentSelectionSet, variableValues, visitedFragments)}.
+      * If {InlineFragment} provides the directive `@defer`, let {deferDirective} be that directive.
+        * If {deferDirective}'s {if} argument is {true} or is a variable in {variableValues} with the value {true}:
+          * Let {deferredFragment} be the result of calling {DeferFragment(objectType, objectValue, fragmentSelectionSet, parentPath)}.
+          * Append {deferredFragment} to {deferredFragments}.
+          * Continue with the next {selection} in {selectionSet}.
+      * Let {fragmentGroupedFieldSet} be the result of calling {CollectFields(objectType, fragmentSelectionSet, variableValues, visitedFragments, parentPath)}.
       * For each {fragmentGroup} in {fragmentGroupedFieldSet}:
         * Let {responseKey} be the response key shared by all fields in {fragmentGroup}.
         * Let {groupForResponseKey} be the list in {groupedFields} for
           {responseKey}; if no such list exists, create it as an empty list.
         * Append all items in {fragmentGroup} to {groupForResponseKey}.
   * Return {groupedFields}.
+  
```

Comment:
Also here, there are two spaces which lead to the note being pulled up on build:

<img width="235" alt="image" src="https://user-images.githubusercontent.com/9714350/93991630-cde7f100-fd8c-11ea-81d7-2c6ba15e4d13.png">